src/Request.php

```diff
@@ -522,4 +529,40 @@ public function setOutputFile($file)
 
         return $this;
     }
+
+    /**
+     * Generates unique alias for the content and stores it in the content map
+     * @param $name string
+     * @return string
+     */
+    private function generateContentAlias($name)
```

Comment:
The method does two things at once. That's better to be avoided i.e. remove `$this->addContentMap($name, $alias);` from the method and call it explicitly.